myRental/test/testUserRequest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If this is not used more than once, we can probably warrant keeping it inside of the test file

Reply:
I figured in the future we might keep it clean by storing json files or expanding that file to have more tests. It might also get removed and replaced with a faker.